frame/evm/test-vector-support/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "pallet-evm-test-vector-support"
+version = "1.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "Apache-2.0"
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
```

Comment:
```suggestion
repository = "https://github.com/paritytech/frontier/"
```

Reply:
This would actually be the first time this repo uses `frontier` in its `repository`. Want me to update all occurrences? In this PR? Is there a suitable `homepage` to use?

Reply:
Set homepage to https://paritytech.github.io/frontier/ if you want!

Yeah `repository` field should be `https://github.com/paritytech/frontier/`

Reply:
Well, not the first time -- it's inconsistent. I'm happy to clean it up in this PR or another.

```
find | grep Cargo.toml | xargs grep repository
./template/runtime/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./template/node/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./primitives/consensus/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./primitives/evm/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./client/consensus/Cargo.toml:repository = "https://github.com/paritytech/frontier/"
./client/db/Cargo.toml:repository = "https://github.com/paritytech/frontier/"
./frame/evm/test-vector-support/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./frame/evm/precompile/ed25519/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./frame/evm/precompile/bn128/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./frame/evm/precompile/simple/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./frame/evm/precompile/dispatch/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./frame/evm/precompile/sha3fips/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./frame/evm/precompile/blake2/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./frame/evm/precompile/modexp/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
./frame/evm/Cargo.toml:repository = "https://github.com/paritytech/substrate/"
```

Reply:
You can clean it up if you want, or we can fix it in a separate PR!

Reply:
065215d